tests/network/primary_pod_network.go

```diff
@@ -52,7 +52,10 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			By("Making sure IP reported on the VMI matches the one on the pod")
 			Expect(vmi.Status.Interfaces[0].IP).To(Equal(vmiPod.Status.PodIP))
 			Expect(vmi.Status.Interfaces[0].IPs).NotTo(BeEmpty())
-			Expect(vmi.Status.Interfaces[0].IPs[0]).To(Equal(vmiPod.Status.PodIP))
+			Expect(len(vmi.Status.Interfaces[0].IPs)).To(Equal(len(vmiPod.Status.PodIPs)))
```

Comment:
I think you can use here the new helper as well.

Reply:
right, on it, thx

Reply:
right, Done, thx